plugin_example/account_setup/main.py

```diff
@@ -0,0 +1,38 @@
+class account_setup_plugin:
```

Comment:
We need to store somewhere DUNE instance.
I tried using i.e. global variable, but it gives warning in pylint.
In the end it seems below version (with static variable in class) is a compromise.

Reply:
Is storing it as a global better? We can certainly disable pylint for that line if needed.

Alternatively, we could add a `common.py`  (or similar) file to `src/pluggin/` that can point us to the DUNE instance, right?

Reply:
> Is storing it as a global better? We can certainly disable pylint for that line if needed.

I think it would be easier to read for users. But they would be surprised with pylint disabling, so I prefer current version.

> Alternatively, we could add a `common.py` (or similar) file to `src/pluggin/` that can point us to the DUNE instance, right?

Currently object of dune (dune_sys) is stored in __main__.py. It would need to be refactored to have some kind of singleton for DUNE. Probably it's not worth to do it.